codegenerator/cli/README.md

```diff
@@ -1,33 +1,47 @@
 
-Designed with a core focus on delivering superior performance and seamless developer experience, Envio optimizes the user experience.
+HyperIndex is a fast, developer-friendly multichain indexer, optimized for both local development and reliable hosted deployment. It empowers developers to effortlessly build robust backends for blockchain applications.
 
-Build a real-time API for your blockchain application in minutes.
+> For comprehensive documentation, visit our [Documentation Portal](https://docs.envio.dev).
+> For hosted services and other solutions, visit [envio.dev](https://envio.dev).
 
-> 📖 For a thorough understanding and to dive deeper into each feature, refer to the original [documentation website](https://docs.envio.dev).
+![Envio Sync Gif](./sync.gif)
 
 ## Key Features
 
-- Simple installation & Setup
-- Any EVM blockchain, as well as Fuel
-- The fastest historical sync
-- Real-time indexing with reorg handling
-- Multi-chain support
-- Write JavaScript, TypeScript, or ReScript with automatically generated types
-- Detailed logging & Error messaging
-- [Hosted Service](https://docs.envio.dev/docs/HyperIndex/hosted-service) to take care of your infrastructure
-- Seamlessly integrate new chains and enhance reliability with the RPC data source
+- **Quickstart templates** – Rapidly bootstrap your indexer
+- **Real-time indexing** – Instantly track blockchain events
+- **Multichain indexing** – Support for multiple blockchains simultaneously
+- **Local development** – Full-featured local environment with Docker
+- **Reorg support** – Graceful handling of blockchain reorganizations
+- **GraphQL API** – Easy-to-query indexed data
+- **Cross-platform support** – Index any EVM-compatible blockchain and Fuel
+- **High performance** – Historical backfills at over 5,000+ events per second
+- **Indexer auto-generation** – Generate indexers directly from smart contract addresses
+- **Flexible language support** – JavaScript, TypeScript, and ReScript
+- **Factory contract support** – Index data from 100,000+ factory contracts seamlessly
+- **On-chain & off-chain data integration** – Easily combine multiple data sources
+- **Self-hosted & managed options** – Run your own setup or use HyperIndex hosted services
+- **Detailed logging & error reporting** – Debug and optimize with clarity
+- **External API actions** – Trigger external services based on blockchain events
+- **Wildcard topic indexing** – Flexible indexing based on event topics
+- **Fallback RPC data sources** – Enhanced reliability with RPC connections
```

Comment:
It would be nice to have links

Reply:
I'd combine `Quickstart templates` and `Indexer auto-generation`

Reply:
Might be worth mentioning our benchmarking and integration testing tools, but this might be too much.